language/move-analyzer/src/symbols.rs

```diff
@@ -431,16 +431,21 @@ impl SymbolicatorRunner {
         cvar.notify_one();
     }
 
+    /// Finds manifest file in a subdirectory of a Move source file passed as argument
     fn root_dir(starting_path: &Path) -> Option<PathBuf> {
+        eprintln!("Looking for manifest file");
```

Comment:
Maybe remove these debug prints?

Reply:
Good point! I thought I'd leave the for future debugging in the field but since these are printed every time a file is changed, it's a bit much.